backend/src/test/java/com/example/backend/auth/api/service/jwt/JwtServiceTest.java

```diff
@@ -0,0 +1,188 @@
+package com.example.backend.auth.api.service.jwt;
+
+import com.example.backend.auth.TestConfig;
+import com.example.backend.common.exception.ExceptionMessage;
+import com.example.backend.common.exception.jwt.JwtException;
+import com.example.backend.domain.define.user.User;
+import com.example.backend.domain.define.user.constant.UserRole;
+import com.example.backend.domain.define.user.repository.UserRepository;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Date;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class JwtServiceTest extends TestConfig {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    JwtService jwtService;
+
+    @AfterEach
+    void tearDown() {
+        userRepository.deleteAllInBatch();
+    }
+
+    @Test
+    @DisplayName("JWT Access 토큰에서 모든 Claims를 추출한다.")
+    void extractAllClaimsTest() {
+        // given
+        User savedUser = userRepository.save(generateUser());
```

Comment:
제가 온보딩 프로젝트하면서 많이 리뷰받았던 부분인데 아래처럼 변경해보면 어떨까요??

```suggestion
        User savedUser = userRepository.save(generateUser());
        String role = savedUser.getRole().name();
        String name = savedUser.getName();
        String profileImageUrl = savedUser.getName();
        ...
```

테스트에 쓰이는 값들을 변수로 정의해놓고, 검증할때 해당 변수로 검증해주면 값이 동적으로 바뀌더라도 테스트가 깨지지 않아요 ㅎㅎ